src/main/java/jewellery/inventory/mapper/OrganizationMapper.java

```diff
@@ -26,4 +33,51 @@ public OrganizationResponseDto toResponse(Organization organization) {
     organizationResponseDto.setNote(organization.getNote());
     return organizationResponseDto;
   }
+
+  private List<UserInOrganizationResponseDto> toUserInOrganizationResponseDtoResponse(
```

Comment:
I would argue we should just call all the methods in the mapper `map` but for now let's at least remove the redundant Reponse. 
Edit: This is also a list... But the suggestion looks bad. I am leaning towards `map` more and more 🤭 
```suggestion
  private List<UserInOrganizationResponseDto> toUserInOrganizationListResponseDto(
```